src/coreclr/tools/Common/InstructionSetHelpers.cs

```diff
@@ -31,7 +31,7 @@ public static InstructionSetSupport ConfigureInstructionSetSupport(string instru
 
             if ((targetArchitecture == TargetArchitecture.X86) || (targetArchitecture == TargetArchitecture.X64))
             {
-                if (isReadyToRun && (targetOS != TargetOS.OSX))
+                if (isReadyToRun && ((targetOS == TargetOS.OSX) || (targetOS == TargetOS.MacCatalyst)))
```

Comment:
Is it correct? Before linux/windows would get `"x86-64-v3"`, now `"x86-64-v2"`.

Reply:
Yeah I flipped the logic while iterating on this. Good catch.
```suggestion
                if (isReadyToRun && targetOS != TargetOS.OSX && targetOS != TargetOS.MacCatalyst)
```